docs/components/motor/dmc4000.md

```diff
@@ -105,6 +115,7 @@ The following attributes are available for `DMC4000` motors:
 | `dir_flip` | bool | Optional | Flip the direction of the signal sent if there is a DIR pin. |
 | `max_rpm` | number | Optional | Set a limit on maximum revolutions per minute that the motor can be run at. |
 | `max_acceleration_rpm_per_sec` | number | Optional | Set the maximum revolutions per minute (RPM) per second acceleration limit. |
+| `control_parameters` | object | Optional | A JSON object containing the coefficients for the proportional, integrative, and derivative terms. If you use `{ "p": 0, "i": 0, "d": 0 }`, `viam-server` auto-tunes the values and logs the values. Tuning takes several seconds and spins the motors. To avoid tuning every time the robot starts up, copy the values from the log and add them to the configuration once tuned. For more information see [Control motor velocity with a sensor](#control-motor-velocity-with-a-sensor). |
```

Comment:
```suggestion
| `control_parameters` | object | Optional | A JSON object containing the coefficients for the proportional, integral, and derivative terms for linear and angular velocity. If you want these values to be auto-tuned, you can set all values to 0: `[ { "type": "linear_velocity", "p": 0, "i": 0, "d": 0 }, { "type": "angular_velocity", "p": 0, "i": 0, "d": 0 } ]`, and `viam-server` will auto-tune and logs the calculated values. Tuning takes several seconds and spins the motors. To avoid tuning every time the robot starts up, copy the values from the log and add them to the configuration once tuned. For more information see [Control motor velocity with a sensor](#control-motor-velocity-with-a-sensor). |
```

Reply:
also removed